server.js

```diff
@@ -21,8 +21,8 @@
 const http = require('http');

```

Comment:
Consider using a constant variable name in uppercase for 'PORT' to follow common naming conventions for constants.